modules/utils.nf

```diff
@@ -24,4 +24,28 @@ def filepath_matches_chr_prefix(fp, chr_prefix){
 def leave_chr_out(chr_prefix, bed_files){
     def bed_files_not_matching_chr_prefix = bed_files.findAll{ fp -> !filepath_matches_chr_prefix(fp, chr_prefix) }
     return [chr_prefix, bed_files_not_matching_chr_prefix]
+}
+
+def processEstimatorsConfig(configValue) {
+    // If it's a list, take the first element (ex: in NULL simulation test config)
+    if (configValue instanceof List) {
+        if (configValue.size() == 0) {
+            error "ESTIMATORS_CONFIG list is empty"
+        }
+        configValue = configValue[0]
+        configValue = configValue.replaceAll(/^\[|\]$|"/, '') // remove brackets if present
+    }
+
+    def configFile = file(configValue)
+
+    // If it's not an existing file, create an empty file with this name
+    if (!configFile.exists()) {
+        configFile = file("${launchDir}/${configValue}")
+        configFile.text = '' // Create an empty file
+        println "Created empty estimators config file: ${configFile}"
```

Comment:
Does that print back to the user? I'm afraid this might lead to some confusion.

Reply:
I can remove the print statement, that is fine!